Trimble.Modus.Components/Controls/TMAccordion/TMAccordion.xaml.cs

```diff
@@ -134,7 +134,13 @@ private async void Close()
     /// Opening the accordion
     /// </summary>
     private async void Open()
-    {
+    {
+        if (this.Parent.Parent is ListView)
+        {
+#if IOS || MACCATALYST
+			throw new NotSupportedException($"{nameof(TMAccordion)} is not yet supported in {Parent.Parent.GetType().Name}");

```

Comment:
Can you include the following in the message?

TMAccordion is not yet supported in ListView. Please use the Bindable Layout.